labs/a1.md

```diff
@@ -116,5 +116,38 @@ ab/abizer/some/path
 
 Hint: The easiest way to do this is with one very short command.
 
+## Question 4
+
+At some point, everyone has looked at a problem and thought to themselves:
+"Hey, I can do this in one line!"
+
+Lets find out if you can. I need to sort out some of my most listened to albums
+by making directories for each of their songs, specifically for my favorite artist Future.
+
+You need to fetch the list of albums and their respective artist from
```

Comment:
I think it would be good to clarify if you expect people to download the file separately and use `cat`, or use `wget`/`curl` in the solution.